db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepository.kt

```diff
@@ -162,7 +153,7 @@ class InntektsmeldingRepository(
     private fun hentNyesteImSkjemaQuery(forespoerselId: UUID): Query =
         InntektsmeldingEntitet
             .selectAll()
-            .where { (InntektsmeldingEntitet.forespoerselId eq forespoerselId.toString()) and InntektsmeldingEntitet.skjema.isNotNull() }
+            .where { InntektsmeldingEntitet.forespoerselId eq forespoerselId.toString() }
```

Comment:
Dette må gjøres for å støtte en slik situasjon:
1. AG sender inn via nav.no
2. AG sender inn via LPS
3. AG sender inn via nav.no, hvor IM er duplikat av 1 (skal være lov)